ios/FluentUI.Demo/FluentUI.Demo/Demos/PillButtonBarDemoController.swift

```diff
@@ -46,7 +46,7 @@ class PillButtonBarDemoController: DemoController {
         disablePrimarySwitchView.addTarget(self, action: #selector(togglePrimaryPills(switchView:)), for: .valueChanged)
 
         container.addArrangedSubview(createLabelWithText("Primary"))
-        addRow(items: [createLabelWithText("Enable/Disable pills in Primary Pill bar"), disablePrimarySwitchView], itemSpacing: 20, centerItems: true)
+        addSectionToggle(toggleTitle: createLabelWithText("Enable/Disable pills in Primary Pill bar"), switchView: disablePrimarySwitchView)
```

Comment:
I think we should let this expand instead of truncate the text.
I know it's out of the scope of the bug fix, but it'd be a nice upgrade.

Reply:
Sure, I can look in to that in a separate PR